.travis.yml

```diff
@@ -22,9 +22,30 @@ install:
   - source ~/.bashrc
   - git clone https://github.com/ethereum/go-ethereum
   - sed -i 's/ctx, cancel = context\.WithTimeout(ctx, time\.Second\*5)/ctx, cancel = context\.WithTimeout(ctx, time\.Second\*500)/' ./go-ethereum/internal/ethapi/api.go
-    
-  - travis_wait 40 make -C go-ethereum geth
+  - travis_wait 40 make -C go-ethereum
   - sudo cp ./go-ethereum/build/bin/geth /usr/local/bin/geth
+  - stty tostop
+  - sudo iptables -L
+  - sudo iptables -A INPUT -p tcp --dport 8545 -j ACCEPT
```

Comment:
OMG this is really required here?

Reply:
no but it was a really painful geth setup. the ubuntu image travis gives out is quite minimalistic. at a certain point i suspected there was something wrong with the iptables. i left some debug stuff inside the travis build script so maybe in the future when things change, they wont be as painful.